File: 0014-longest-common-prefix/0014-longest-common-prefix.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        Array.Sort(strs); 
        Console.WriteLine(string.Join(", ", strs));
        string first = strs[0];
        string last = strs[^1];
        string x = "";
        int mini = Math.Min(first.Length, last.Length);
        for (int i = 0 ; i < mini; i++) {
            if (first[i] == last[i]) { 
                x = x + first[i];
            }
            else { 
                break;
            }

        }
return x;        
        
    }
}

File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        s = sorted(strs) 
        first = s[0]
        last = s[-1] 
        res = "" 
        for i in range(min(len(first), len(last))): 
            if first[i] == last[i]: 
                res = res + first[i]
            else : 
                break 
        return res

        
        